datacube_stats/main.py

```diff
@@ -109,19 +111,33 @@ def gather_tile_indexes(tile_index, tile_index_file):
               help="A file consisting of tile indexes specified as [X] [Y] per line")
 @click.option('--output-location', help='Override output location in configuration file')
 @click.option('--year', type=int, help='Override time period in configuration file')
+@click.option('--task-slice', type=str, default='::',
+              help="The subset of tasks to perform, using Python's slice syntax.")
+@click.option('--batch', type=int,
+              help="The number of batch jobs to launch using PBS and the serial executor.")
 @click.option('--list-statistics', is_flag=True, callback=list_statistics, expose_value=False)
 @ui.global_cli_options
 @with_qsub_runner()
 @click.option('--version', is_flag=True, callback=_print_version,
               expose_value=False, is_eager=True)
 @ui.pass_index(app_name='datacube-stats')
 def main(index, stats_config_file, qsub, runner, save_tasks, load_tasks,
-         tile_index, tile_index_file, output_location, year):
+         tile_index, tile_index_file, output_location, year, task_slice, batch):
     if qsub is not None:
-        # TODO: verify config before calling qsub submit
-        click.echo(repr(qsub))
-        exit_code, _ = qsub(auto=True)
-        return exit_code
+        if batch is None:
```

Comment:
I don't like that `is not None` is followed immediately by `is None`. Maybe

```python
if qsub and batch:
  ...
elif qsub:
  ...
```
as much as I dislike default truth checking